package.json

```diff
@@ -9,9 +9,9 @@
     "test:watch": "node node_modules/.bin/jest --watchAll",
     "test:current": "node node_modules/.bin/jest --watch",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "storybook": "start-storybook -p 6006",
-    "storybook:build": "node --max_old_space_size=2048 ./node_modules/.bin/build-storybook --config-dir .storybook --output-dir .storybook-dist",
-    "storybook:deploy": "surge --project .storybook-dist",
+    "stories": "tfm-stories --plugin",
+    "stories:build": "tfm-build-stories --plugin",
+    "stories:deploy": "surge --project .storybook-dist",
```

Comment:
does it make sense to build it first?

Reply:
I would want to deliver an opinionated way to do deploy in the next version and I got some cool Ideas (Github Actions).
This is an attempt to keep the current things work.